delta/rdf/src/test/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/JsonLdEncoderSpec.scala

```diff
@@ -66,6 +66,11 @@ class JsonLdEncoderSpec extends AnyWordSpecLike with Matchers with Fixtures with
         result.toString should equalLinesUnordered(ntriples(result.rootNode.asBNode.value))
       }
 
+      "return a NQuads format" in {
+        val result = permissions.toNQuads.accepted
+        result.toString should equalLinesUnordered(ntriples(result.rootNode.asBNode.value))
```

Comment:
the representation is identical to the ntriples in this case; can we also have a test with a named graph that shows the quads?
wouldn't the n-quads show a bnode in the graph position if there an anonymous named graph?

Reply:
No, that's not the representation that gives back Jena, which seems fine to me. Again, to see the real test, check on the GraphSpec. There you can see an example where you actually have a namedgraph in place.

Reply:
https://github.com/BlueBrain/nexus/blob/d190c3bd9ee7886ed9f6f7600b441dcad566d6b7/delta/rdf/src/test/resources/graph/multiple-roots-namedgraph.nq

Reply:
I'm aware that if there's no graph the representation is identical to the triples. However, if the graph doesn't have an `@id`, the n-quads representation should show a blank node in the graph position. For example, the json-ld playground show for:
```
{
  "@context": {
    "@vocab": "https://schema.org/",
    "@base": "https://schema.org/"
  },
  "@graph": [
    {
      "@id": "Person",
      "@type": "Property"
    }
  ],
  "deprecated": true
}
```
this representation:
```
<https://schema.org/Person> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/Property> _:b0 .
_:b0 <https://schema.org/deprecated> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
```

Reply:
and?
1) That's not what you have on the tests.
2) We don't need to test for that use case since we never have it.

Reply:
1. it's good to cover this functionality if it's exposed as a public method in a class, even if it makes use of another class.
2. in the case of `/source` with an `@graph` and automatically generated id with user defined properties, wouldn't this functionality be accessed?
3. this test does not exist in GraphSpec

Reply:
1. That's why the current test is enough.
2. The `/source` endpoint is to get the source as is. We should not allow that endpoint to try to do any conversions, otherwise it defeats the point
3. in `GraphSpec` there is a test for converting a graph to nquads.